build-image/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM docker.io/bitnami/minideb:bullseye-amd64 as slimmed-rust
```

Comment:
Given this is just a builder, is there a specific reason not to use just the official debian image? If yes, it'd be nice to drop a comment

Reply:
if @SkymanOne took that from the Dockerfile I linked earlier then I should explain - I chose it b/c of active development of the relevant repository and reputation of the organization that's owning it https://github.com/bitnami/minideb

Reply:
Yes. I took your dockerfile and minideb has better security policy and smaller size even than slim version of Debian.